lldb/source/Plugins/Language/Swift/FoundationValueTypes.cpp

```diff
@@ -30,14 +30,23 @@ using namespace lldb_private;
 using namespace lldb_private::formatters;
 using namespace lldb_private::formatters::swift;
 
+static constexpr bool show_unavailable_workaround = true;
+static const char *unavailable_field_workaround =
+  "<unavailable; try printing with \"v -O\" or \"po\">";
```

Comment:
There's a `vo` alias, should that be used?

Reply:
Definitely — I forgot that exists.
I've also debated whether I should spell this out as `frame variable -O` but I thought this is probably not the right place to educate users.